.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 # HOWTO: https://pre-commit.com/#usage
 # pip3 install pre-commit
-# pre-commit install
```

Comment:
Can you please update config in other repositories as well, please?

Reply:
yes